### PR TITLE
Convert function

### PR DESCRIPTION
**Pull request by francoiseblanc**
This is the first version of the _convert_results_ function which converts Cadmium Cell-DEVS results (i.e. .txt file and .json file) in the required format for the viewer. An example of results files to be converted have been placed into: ../data/results.

### CODE REVIEW

File: include/simulation_lifecycle/error.h [hunk 1]
```diff
@@ -30,4 +30,11 @@
 /** VISUALIZATION ERRORS 400-499 */
 // TODO
 
+/** CONVERSION ERRORS 500-599 */
+#define INPUT_PATH_EMPTY 500
+#define INPUT_PATH_FOLDER_ERROR 501
+#define FILE_FORMAT_INCORRECT 502
```
Comment (romancardenas) [inline include/simulation_lifecycle/error.h:36]: Can you add the `CONVERT` prefix to all the errors? In this way, we can know that the error has to do with `CONVERT`

File: test/test_convert.c [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+/**
+ *  Function convert_results - Test case #1
+ *  Description: Call the function with path being null.
+ *  Expected output: The function returns 340.
+ */
+void test_function_convert_results_test_case_1(void) {
+    char *results_folder_path = NULL;
+    int error_code = convert_results(results_folder_path);
+    TEST_ASSERT_EQUAL(error_code, INPUT_PATH_EMPTY);
```
Comment (romancardenas) [inline test/test_convert.c:21]: normally, you use `TEST_ASSERT_EQUAL(expected, actual)`.
It is not a big deal, but I suggest you change this to `TEST_ASSERT_EQUAL(INPUT_PATH_EMPTY, error_code)`.
The same for the other tests

**Review by romancardenas — APPROVED** [review]
Nice work! Please, check my comments

File: include/simulation_lifecycle/error.h (hunk 1 above)
Comment (francoiseblanc) [reply include/simulation_lifecycle/error.h]: Thank you @romancardenas for the review! I've made the changes and just pushed a new commit.

File: test/test_convert.c (hunk 2 above)
Comment (francoiseblanc) [reply test/test_convert.c]: Changes have been made in the new commit :) Thanks!

**Review by ghidadandan — APPROVED** [review]
Looks good Francoise

**Pull request merged**